build-tools/scripts/TestApks.targets

```diff
@@ -169,16 +171,18 @@
   <Target Name="DeployTestAabs"
       Condition=" '@(TestAab)' != '' ">
     <PropertyGroup>
-      <_KeyStore>..\src\Xamarin.Android.Build.Tasks\Tests\Xamarin.ProjectTools\Resources\Base\test.keystore</_KeyStore>
+      <_KeyStore>$(XamarinAndroidSourcePath)\src\Xamarin.Android.Build.Tasks\Tests\Xamarin.ProjectTools\Resources\Base\test.keystore</_KeyStore>
       <_KeyAlias>mykey</_KeyAlias>
       <_KeyPass>android</_KeyPass>
       <_StorePass>android</_StorePass>
+      <JavaPath Condition=" '$(JavaPath)' == '' And !$([MSBuild]::IsOsPlatform(Windows)) ">$(JavaSdkDirectory)\bin\java</JavaPath>
+      <JavaPath Condition=" '$(JavaPath)' == '' And $([MSBuild]::IsOsPlatform(Windows)) ">$(JavaSdkDirectory)\bin\java.exe</JavaPath>
     </PropertyGroup>
     <Xamarin.Android.Tools.BootstrapTasks.BundleTool
         Arguments="build-apks --connected-device --overwrite --mode default --bundle &quot;%(TestAab.Identity)&quot; --output &quot;%(TestAab.Identity).apks&quot;"
         ContinueOnError="ErrorAndContinue"
         JavaPath="$(JavaPath)"
-        JarPath="$(BundleToolJarPath)"
+        JarPath="$(AndroidBundleToolJarPath)"
```

Comment:
@jonathanpeppers I haven't thoroughly tested this change. I think it's safe WRT "in-build-tree" testing given that `xabuild` overwrites `MonoAndroidToolsDirectory`, but let me know if any concern immediately comes to mind. I don't want this to accidentally prefer the `AndroidBundleToolJarPath` version on disk when trying to test against build tree changes rather than a system installation, so a different approach may be warranted.

Reply:
I think `$(BundleToolJarPath)` points to a place in-tree:

https://github.com/xamarin/xamarin-android/blob/9de47c597c6f653be5b4a1f23e9b4d35e8bf7d70/Configuration.props#L143

The way this will work, will the test run set `$(XAInstallPrefix)`?

https://github.com/xamarin/xamarin-android/blob/9de47c597c6f653be5b4a1f23e9b4d35e8bf7d70/Configuration.props#L44

Reply:
I've updated this to continue to use the default `<BundleToolJarPath/>` value unless the file does not exist. If the file does not exist, we'll use `<AndroidBundleToolJarPath/>` instead.